packages/app/src/WalletDropdown.tsx

```diff
@@ -1,7 +1,8 @@
 import React, { useContext, useEffect, useState } from 'react';
 import { Text, View, StyleSheet } from 'react-native';
 import { SafeAreaView } from 'react-native-safe-area-context';
-import { graphql, useFragment, useMutation } from 'relay-hooks';
+import { graphql, useMutation } from 'relay-hooks';
```

Comment:
please use `graphql` from `react-relay` so we can deprecate our reliance on `relay-hooks`